### PR TITLE
orchis: init at 2021-01-22

### PR DESCRIPTION
**Pull request by fufexan**
[Orchis](https://github.com/vinceliuice/Orchis-theme) is a Material Derign theme for GTK.

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: maintainers/maintainer-list.nix [hunk 1]
```diff
@@ -3237,6 +3237,12 @@
     githubId = 10528737;
     name = "Severin Fürbringer";
   };
+  fufexan = {
```
Comment (SuperSandro2000) [inline maintainers/maintainer-list.nix:3240]: Please put this into a separate commit with the message ``maintainers: add fufexan``.

File: pkgs/data/themes/orchis/default.nix [hunk 2]
```diff
@@ -0,0 +1,35 @@
+{ accentColor ? "blue"
+, lib, stdenv, fetchFromGitHub, gtk3, gnome-themes-extra, gtk-engine-murrine }:
```
Comment (SuperSandro2000) [inline pkgs/data/themes/orchis/default.nix:2]:
```suggestion
{ lib, stdenv, fetchFromGitHub, gtk3, gnome-themes-extra, gtk-engine-murrine
, accentColor ? "blue" }:
```

options are usually at the end of inputs.

File: pkgs/data/themes/orchis/default.nix [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+  buildInputs = [ gnome-themes-extra ];
+
+  propagatedUserEnvPkgs = [ gtk-engine-murrine ];
+
+  dontPatch = true;
+  dontConfigure = true;
+  dontBuild = true;
+
+  preInstall = "mkdir -p $out/share/themes";
+  installPhase = "bash install.sh -d $out/share/themes" + lib.optionalString (accentColor != "blue") " -t ${accentColor}";
```
Comment (SuperSandro2000) [inline pkgs/data/themes/orchis/default.nix:26]:
```suggestion
  preInstall = ''
    mkdir -p $out/share/themes
  '';

  installPhase = ''
    runHook preInstall
    bash install.sh -d $out/share/themes -t ${accentColor}
    runHook postInstall
  '';
```

Does that work when we always specify the accent color?

File: pkgs/data/themes/orchis/default.nix [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+  meta = with stdenv.lib; {
```
Comment (SuperSandro2000) [inline pkgs/data/themes/orchis/default.nix:28]:
```suggestion
  meta = with lib; {
```

File: pkgs/data/themes/orchis/default.nix [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+    description = "Orchis is a Material Design theme for GNOME/GTK based desktop environments.";
```
Comment (SuperSandro2000) [inline pkgs/data/themes/orchis/default.nix:29]:
```suggestion
    description = "A Material Design theme for GNOME/GTK based desktop environments";
```

File: pkgs/top-level/all-packages.nix [hunk 6]
```diff
@@ -20484,6 +20484,8 @@ in
 
   orion = callPackage ../data/themes/orion {};
 
+  orchis = callPackage ../data/themes/orchis {};
```
Comment (SuperSandro2000) [inline pkgs/top-level/all-packages.nix:20487]:
```suggestion
  orchis = callPackage ../data/themes/orchis { };
```

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

File: maintainers/maintainer-list.nix (hunk 1 above)
Comment (fufexan) [reply maintainers/maintainer-list.nix:3240]: Done.

File: pkgs/data/themes/orchis/default.nix (hunk 3 above)
Comment (fufexan) [reply pkgs/data/themes/orchis/default.nix:26]: I think yes. The default is "blue" and the script works the same whether there's `-t blue` present or not.

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review with [nixpkgs-review-checks extension](https://github.com/SuperSandro2000/nixpkgs-review-checks). It is checked by a human on a best effort basis and does not build all packages (e.g. lumo, tensorflow or pytorch).
If you have any questions or problems please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 111347` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
1 package built:
</summary>
<ul>
<li>orchis</li>
</ul>
</details>

The following issues got detected with the above build packages.
Please fix at least the ones listed with your changed packages:

orchis:
<details>

An avoidable string conversion got detected: `    rev = "${version}";`
Please do not convert variable to a string without adding a string like `v`.

</details>

**Comment by fufexan** [comment]
Would replacing `rev = "${version};"` with `rev = version;` be considered alright?

**Comment by SuperSandro2000** [comment]
> Would replacing `rev = "${version}";` with `rev = version;` be considered alright?

Yeah that totally fine. I should add it to the text.

**Comment by jtojnar** [comment]
Does it work without `sassc`? It is listed in the readme as a dependency.

**Comment by fufexan** [comment]
It installs fine. I think `sassc` is required to compile the `sass` files, but it all seems to work alright even without it.

**Comment by fufexan** [comment]
Can this be merged or do I need to do anything else? @SuperSandro2000
